components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/list-workflows.jsp

```diff
@@ -155,24 +156,24 @@
                 %>
                 <tr>
                     <td>
-                        <a href="view-workflow.jsp?<%=WorkflowUIConstants.PARAM_WORKFLOW_ID%>=<%=workflow.getWorkflowId()%>">
-                        <%=workflow.getWorkflowName()%>
+                        <a href="view-workflow.jsp?<%=WorkflowUIConstants.PARAM_WORKFLOW_ID%>=<%=Encode.forHtmlAttribute(workflow.getWorkflowId())%>">
+                        <%=Encode.forHtmlContent(workflow.getWorkflowName())%>
                         </a>
                     </td>
-                    <td><%=workflow.getWorkflowDescription() == null ? "" : workflow.getWorkflowDescription()%>
+                    <td><%=workflow.getWorkflowDescription() == null ? "" : Encode.forHtmlContent(workflow.getWorkflowDescription())%>
                     </td>
-                    <td><%=workflow.getTemplate().getName()%>
+                    <td><%=Encode.forHtmlContent(workflow.getTemplate().getName())%>
                     </td>
-                    <td><%=workflow.getWorkflowImpl().getWorkflowImplName()%>
+                    <td><%=Encode.forHtmlContent(workflow.getWorkflowImpl().getWorkflowImplName())%>
                     </td>
                     <td>
                         <a title="<fmt:message key='workflow.service.workflow.edit.title'/>"
-                           onclick="editWorkflow('<%=workflow.getWorkflowId()%>');
+                           onclick="editWorkflow('<%=Encode.forJavaScriptAttribute(workflow.getWorkflowId())%>');
```

Comment:
Passed parameter is used in a query string. So encode for uri component and then encode for javascript attribute